client/src/components/App/app.jsx

```diff
@@ -68,27 +83,36 @@ class App extends React.Component {
 
   handleClick(direction){
     if(direction ==="left"){
-      this.setState(prevState =>({
-        currentPage: prevState.currentPage -1
-      }))
+      if(this.state.currentPage >1){
+        this.setState(prevState =>({
+          currentPage: prevState.currentPage -1
+        }))
+      }
     }else{
-      this.setState(prevState =>({
-        currentPage: prevState.currentPage +1
-      }))
+
+      if(this.state.currentPage < 3){
+        this.setState(prevState =>({
+          currentPage: prevState.currentPage +1
+        }))
+      }
+      
     }
     
   }
 
   componentDidMount(){
-    axios.get(`http://localhost:8081/api/suggestions/${this.state.listingId}`)
+
+    axios.get(window.location.protocol + '//' +  window.location.host + `/api/suggestions/${this.state.listingId}`)
       .then(res =>{
+        console.log('got array')
```

Comment:
curious why you add protocol?  i guess it's one less thing that is hardcoded which is always good.

Reply:
yes they have cors middle ware attached, we can take a look at it todya or tomorrow.
For adding a protocol, yeah it was to make it more dynamic because hardcoding it will make it only work for http but not https